MANUAL.md

```diff
@@ -595,28 +585,27 @@ availability sets for things that need HA / fault-tolerance.
 You must, in turn, define the following VM extensions in your
 cloud config:
 
-  1.  `consul_as` - Consul service discovery cluster availability set.
-  2.  `haproxy_as` - HAProxy availability set.
-  3.  `nats_as` - NATS Message Bus cluster availability set.
-  4.  `uaa_as` - UAA nodes availability set.
-  5.  `api_as` - Cloud Controller API nodes availability set.
-  6.  `doppler_as` - Doppler node availability set.
-  7.  `loggregator_tc_as` - Loggregator / Traffic Controller
+  1.  `haproxy_as` - HAProxy availability set.
+  2.  `nats_as` - NATS Message Bus cluster availability set.
+  3.  `uaa_as` - UAA nodes availability set.
+  4.  `api_as` - Cloud Controller API nodes availability set.
+  5.  `doppler_as` - Doppler node availability set.
+  6.  `loggregator_tc_as` - Loggregator / Traffic Controller
```

Comment:
You don't have to renumber these, markdown doesn't care what you use (we could always use `1.`)